src/prpy/util.py

```diff
@@ -1683,8 +1770,7 @@ def GetLinearCollisionCheckPts(robot, traj, norm_order=2, sampling_func=None):
         waypoint = traj.GetWaypoint(i)
         q1 = traj_cspec.ExtractJointValues(waypoint, robot, dof_indices)
         dq = numpy.abs(q1 - q0)
-        max_diff_float = numpy.max( numpy.abs(q1 - q0) / q_resolutions)
-        
+
```

Comment:
My auto-linter often compels me to destroy things gently muttering "Now I am become Death, the destroyer of worlds."

> On Apr 4, 2016, at 11:05 AM, Pras Velagapudi <notifications@github.com> wrote:
> 
> In src/prpy/util.py:
> 
> > @@ -1683,8 +1770,7 @@ def GetLinearCollisionCheckPts(robot, traj, norm_order=2, sampling_func=None):
> >          waypoint = traj.GetWaypoint(i)
> >          q1 = traj_cspec.ExtractJointValues(waypoint, robot, dof_indices)
> >          dq = numpy.abs(q1 - q0)
> > -        max_diff_float = numpy.max( numpy.abs(q1 - q0) / q_resolutions)
> > -        
> > +
> Pre-emptive explanation: this variable was totally unused and my auto-linter compelled me to destroy it.
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
>